utils/firestore.js

```diff
@@ -3010,10 +3009,12 @@ const processTwilioEvent = async (event) => {
             status: event.MessageStatus,
             [`${event.MessageStatus}Date`]: date,
             errorCode: event.ErrorCode || "",
-            errorMessage: event.ErrorMessage || "",
+            errorMessage: event.ErrorMessage || twilioErrorMessages[event.ErrorCode] || "",
         };
 
         await db.collection("notifications").doc(doc.id).update(eventRecord);
+
+        console.log(`Finished updating status for messageSid ${event.MessageSid}:${event.MessageStatus}`)
```

Comment:
remove log

Reply:
No, we don't when there's no errors, because there're so many of messages coming in.
Consider printing out the Sid when error occurs.